AUTHORS

```diff
@@ -68,6 +68,11 @@ year of their first commit. GitHub handle is optional.
 ----
 
 - Jinzhe Zeng (@njzjz)
+
+2024
+----
+
+-  @hsk17
```

Comment:
@hsk17 please add your full name if you want to be acknowledged by it

Reply:
Friedrich Haubensak

thanks :-)

Reply:
BTW:  for the numpy.nan issue, there is also https://github.com/Becksteinlab/numkit/pull/32

Reply:
thanks for the heads up -- will check numkit afterwards, feel free to ping me with @orbeckst to get my attention